### PR TITLE
Merge version 0.1.46 from dev to master

### PR DESCRIPTION
**Pull request by richrd**
Features:
- Command for opening the default config file (`config defaults`)
- Faster input handling. This makes pasting over SSH much better.
- Pip installation instructions

Fixes:
- Hide errors if linting subprocess fails.
- Avoid crashing when running a module fails.

### CODE REVIEW

File: suplemon/main.py [hunk 1]
```diff
@@ -148,24 +149,48 @@ def load(self):
...
-                # (View won't update after switching files, WTF)
-                self.trigger_event_after("mainloop")
-                self.get_editor().resize()
-                self.ui.refresh()
+            self.block_rendering = True
+            got_input = False
+
+            # Run through max 100 inputs (so the view is updated at least every 100 characters)
+            i = 0
+            while i < 100:
```
Comment (twolfson) [inline suplemon/main.py]: This magic 100 should prob be defined as a constant at the top

File: suplemon/modules/config.py [hunk 2]
```diff
@@ -9,7 +11,7 @@ class Config(Module):
     def run(self, app, editor, args):
         if args == "defaults":
             # Open the default config in a new file only for viewing
-            path = app.path + "/config/defaults.json"
+            path = os.path.join(app.path, "config/defaults.json")
```
Comment (twolfson) [inline suplemon/modules/config.py]: There is still a `/` between `config` and `defaults.json`. You prob want:

```python
path = os.path.join(app.path, "config", "defaults.json")
```

Comment (richrd) [inline suplemon/modules/config.py]: Doh, must have done that my eyes half closed.

File: suplemon/main.py (hunk 1 above)
Comment (richrd) [inline suplemon/main.py]: Again, good idea :+1: I was in a bit of a hurry.

**Pull request merged**